sdk/helper/tlsutil/tlsutil.go

```diff
@@ -21,6 +21,7 @@ var TLSLookup = map[string]uint16{
 	"tls10": tls.VersionTLS10,
 	"tls11": tls.VersionTLS11,
 	"tls12": tls.VersionTLS12,
+	"tls13": tls.VersionTLS13,
```

Comment:
This looks great.

If we ended up defaulting to TLS 1.3, we'd want to update lines 103 and 144 as well.